DOCS-11893 updating status (#54)
mulesoft/docs-anypoint-datagraph: modules/ROOT/pages/status-updates.adoc

== Status: Updating

> You can add to and update the unified schema at any time. After you apply changes, the states reads *Updating* and indicates the progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.

Reviewer: ```suggestion
You can add to and update the unified schema at any time. After you apply changes, the status reads *Updating* and indicates the progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```

> You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema enters an “updating” state, and the status indicates the progress and queued updates. This status update is visible to anyone in the business group with permissions for the environment.

Reviewer: Do we need the first "and apply changes"? Does "add to and update" not imply that? Your choice, of course:
```suggestion
You can add to and update the unified schema at any time. After you apply changes, the unified schema enters an `updating` state during which you can view the status of in-progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```

image::datagraph-status-update-in-process.png[Status indicator shows adding API schema to unified schema,65%,65%]

When the unified schema is updating, you can view the new changes in the schema, make additional changes, and apply new changes. However, these changes aren’t available to query until the update is complete.

> As queued updates finish, the status changes to finished.

Reviewer: As with "updating" in Line 7, is this a syntax that the user sees?
Use mono for syntax display; bold for UI labels.

Reviewer: See also Line 17 below.
I've edited for syntax presentation but you know best: Lines 7, 13, and 17.

image::datagraph-status-update-finished.png[Status indicator shows an update is finished,65%,65%]

> When all updates complete, the status will change to *Up to date*.

Reviewer: ```suggestion
When all updates are complete, the status changes to *Up to date*.
```
